### PR TITLE
Update package versions and fix issue with script

### PR DESCRIPTION
**Pull request by bonifazi-r3**
Update IntelliJ and CLion versions used inside the container.
Also update the versions of some packages and fix an issue with a script

### CODE REVIEW

File: conclave-enclave/build.gradle [hunk 1]
```diff
@@ -13,5 +13,4 @@ dependencies {
     testImplementation project(":internal-testing")
     testImplementation "org.assertj:assertj-core:$assertj_version"
     testImplementation "org.junit.jupiter:junit-jupiter:$junit_jupiter_version"
-    testImplementation "org.mockito:mockito-inline:$mockito_version"
```
Comment (bonifazi-r3) [inline conclave-enclave/build.gradle:16]: This is unused?

File: conclave-enclave/src/test/kotlin/com/r3/conclave/enclave/internal/EnclaveUtilsTest.kt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.r3.conclave.enclave.internal
 
-import com.r3.conclave.internaltesting.threadWithFuture
```
Comment (bonifazi-r3) [inline conclave-enclave/src/test/kotlin/com/r3/conclave/enclave/internal/EnclaveUtilsTest.kt:3]: Unused

File: integration-tests/general/filesystem-db-enclave/build.gradle [hunk 3]
```diff
@@ -9,7 +9,7 @@ dependencies {
     //  Using 1.18.2 version for JTS triggers a bug, we are using 1.16.0 (see versions.gradle)
     runtimeOnly "org.locationtech.jts:jts:$locationtech_jts_version"
     runtimeOnly "org.locationtech.jts:jts-core:$locationtech_jts_version"
-    runtimeOnly "org.locationtech.spatial4j:spatial4j:$locationtech_spatial4js_version"
+    runtimeOnly "org.locationtech.spatial4j:spatial4j:$locationtech_spatial4j_version"
```
Comment (bonifazi-r3) [inline integration-tests/general/filesystem-db-enclave/build.gradle:12]: The library is called `spatial4j`, I have renamed as it is annoying when searching online

**Comment by bonifazi-r3** [comment]
@shamsasari  I wanted to update my IntelliJ in the container and I have found out that many versions of the packages need update.
I only updated the ones which weren't causing issues, and a minor script fix issue. Clearly not urgent, did this in 20 mins

**Review by shamsasari — APPROVED** [review]

**Pull request merged**